app/static/js/slcNewProposal.js

```diff
@@ -86,17 +86,42 @@ function fixStepIndicator(navigateTab) {
 
 // TODO: empty the courseInstructor input after an instructor has been added to the table.
 function callback() {
-  let instructor = $("#courseInstructor").val();
+  let data = $("#courseInstructor").val();
+  data = data.split(",");
+  let instructor = data[0]
+  let username = instructor.split('(').pop().split(')')[0]; // returns 'two'
+  let phone = data[1]
   let tableBody = $("#instructorTable").find("tbody");
   let lastRow = tableBody.find("tr:last");
   let newRow = lastRow.clone();

```

Comment:
you young'uns and your `let`s. Back in my day we only had `var`, and sometimes we didn't even use that!

Reply:
Don't hate the player hate the game